### PR TITLE
Turn on jobseeker accounts

### PR DESCRIPTION
**Pull request by cpjmcquillan**
Prepare for lift off 🚀

### CODE REVIEW

File: spec/system/hiring_staff_can_manage_their_vacancies_spec.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
     visit root_path
 
-    within("div.manage-vacancies") do
+    within(".govuk-header__navigation") do
```
Comment (alexbowen) [inline spec/system/hiring_staff_can_manage_their_vacancies_spec.rb:11]: 👍

**Review by alexbowen — APPROVED** [review]

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2785.london.cloudapps.digital>

File: spec/components/shared/navbar_component_spec.rb [hunk 2]
```diff
@@ -34,6 +32,10 @@
   end
 
   context "when JobseekerAccountsFeature is not enabled" do
```
Comment (david-mears-dfe) [inline spec/components/shared/navbar_component_spec.rb:34]: Can we remove this context?

Comment (david-mears-dfe) [reply spec/components/shared/navbar_component_spec.rb:34]: And any similar ones

Comment (cpjmcquillan) [reply spec/components/shared/navbar_component_spec.rb:34]: Generally we do that once the feature flag is removed

**Review by csutter — CHANGES_REQUESTED** [review]
As discussed, we're holding off for now 😢

**Review by csutter — APPROVED** [review]

**Pull request merged**